sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/releases_v1/bigconfig.yml

```diff
@@ -0,0 +1,17 @@
+type: BIGCONFIG_FILE
+table_deployments:
+- deployments:
+  - fq_table_name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.org_mozilla_fenix_derived.releases_v1
```

Comment:
Is `moz-fx-data-shared-prod` being here double intentional?

Reply:
Yeah, so Bigeye is doing something really strange where the dataset name is actually `moz-fx-data-shared-prod.<dataset_name>`. I'll bring it up with them in a ticket